newtab.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <!-- viewport meta tag -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- pull in local jquery file -->
+    <script src="thirdparty/jquery-3.4.1.min.js" charset="utf-8"></script>
+    <!-- run app script -->
+    <script src="app.js" charset="utf-8"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>HomeTab</title>
+  </head>
+  <body>
+    <div class="grid-container">
+      <div class="leftMenu">
+        
+      </div>
+      <div class="main">
+        <h1>Random Quote</h1>
+        <div class="retrievedQuote">
+          <p id="quoteText"></p>
+          <h3 id="quoteAuthor"></h3>
+        </div>
+      </div>
+      <div class="rightMenu">
+        <h1>Common Links</h1>
+        <ul class="linkList">
+          <li><a href="https://www.google.com/">Google</a></li>
+          <li><a href="https://www.usagundamstore.com/">Gunpla</a></li>
+          <li><a href="https://twitter.com/home">Twitter</a></li>
```

Comment:
In case you were planning to open the links in a new tab you could also add following attribute to your a tag:
`<a href="https://www.google.com/" target="_blank">Google</a>`

Reply:
Thank you! I honestly wasn't even thinking about that.